tests/PasswordGeneratorTest.php

```
@@ -0,0 +1,240 @@
+            1,
+            preg_match('/[!@#$%^&*()_\-=+;:,.?]/', $password),
+            'Should include at least one special character.',
+        );
+    }
+
+    public function testGenerateCharacterDistributionIncludesAllCharacterGroups(): void
+    {
+        $buffer = '';
+
+        for ($i = 0; $i < 10; $i++) {
+            $buffer .= PasswordGenerator::generate(50);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Helper\PasswordGenerator' in method 'testGenerateCharacterDistributionIncludesAllCharacterGroups'. (undefined)

(StaticAccess)